sqlmesh/core/schema_diff.py

```diff
@@ -687,5 +687,14 @@ def has_drop_alteration(alter_expressions: t.List[exp.Alter]) -> bool:
     )
 
 
+def get_dropped_column_name(alter_expressions: t.List[exp.Alter]) -> t.Optional[str]:
```

Comment:
Should we print all impacted columns at once?

Reply:
Definitely, I should have thought of that